jekyll.yml

```diff
@@ -212,6 +212,8 @@ mesh_raw_generated_paths:
   - app/_src/.repos/kuma/app/docs
 # Helm commands
 set_flag_values_prefix: kuma.
+mesh_helm_repo_url: https://kong.github.io/kong-mesh-charts
+mesh_helm_repo_name: kong-mesh
```

Comment:
@lahabana can you add these to `jekyll-dev.yml` too?

Reply:
Done!